CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Changelog
 
-This file is currently only a template for future use.
-
-Each change falls into one of the following categories: Added, Changed, Deprecated, Removed, Fixed or Security.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.2.0] - 2022-12-08
+
 ### Changed
  - Enable modules schnorrsig, extrakeys and ECDH by default in ./configure
```

Comment:
```suggestion
 - Enabled modules schnorrsig, extrakeys and ECDH by default in ./configure.
```
nit: seems that we use past tense everywhere else (and this fits the headings), so we could fix this in this PR.

Reply:
Fixed (the example at keepachangelog.com uses both present and past tense)